dashboard/app/models/ability.rb

```diff
@@ -233,7 +234,7 @@ def initialize(user)
     if user.persisted?
       editor_experiment = Experiment.get_editor_experiment(user)
       if editor_experiment
-        can :clone, Level
+        can :clone, Level, &:custom?
         can :manage, Level, editor_experiment: editor_experiment
```

Comment:
Editor experiment users get the ability to index Level on this line.  Note that our index route doesn't actually use the autoloaded `@levels` so they'll be able to see all levels, not just those they have permission to manage.  It might be cleaner to add `can :index, Level` here and actually use the autoloaded levels in that route, which would let us easily restrict levels listed on that page via abilities later.

Reply:
Thanks for tracking this down! It makes some amount of sense that this line would enable partners to `:index`, so I don't feel strongly about whether to further clarify.

It is unclear to me at the moment whether partners should be able to see all levels or just the ones they own, because they may want to clone levels they do not own. I think it's fine for now for them to see all levels. If you do build a feature to filter by ownership, perhaps we can extend it later to work for platformization partners if needed.

Reply:
Update: Actually went ahead and did this.